AUTHORS.md

```diff
@@ -1 +1,32 @@
-FIXME: list authors' names and email addresses.
\ No newline at end of file
+Thank you to the following contributors (in alphabetical order by user name):
```

Comment:
I think the order is not being followed in the doc.

Reply:
Good catch, I have fixed that